### DISCUSSION TITLE
Model Viewer and GDrive

### DISCUSSION THREAD

**Discussion by alsanxe** [google/model-viewer#3530] [Q&A]
It is possible to get model viewer with a glb file uploaded in Google Drive? I also want to know where you upload your files to use with model viewer, i've tried firebase but i'm managing with big glb files (300mb) and bandwith consumption is too huge.

Thanks!

**Comment by elalish** [comment]
Why are your files 300mb? It sounds like you may need to look into decimation and compression for web delivery. I don't believe Drive allows serving GLBs currently. We're using Github pages, but our models are <10mb. You can use Glitch too but it might be slow.
